### ISSUE TITLE
Scroll bars no longer function in Admin Menu

### ISSUE THREAD

**Issue by Rusty01** [Person8880/Shine#770]
Scroll bars no longer work for the admin menu
So if you click bans and want to scroll down, it won't move, You can't use your mousewheel either.

**Comment by Person8880** [comment]
The bans list shouldn't have a scrollbar, it's paginated using the buttons at the bottom. Is there a scrollbar visible in the list? Are there other lists that aren't working? All the scrollbars that show up when I test locally seem to work fine.

**Comment by Rusty01** [comment]
You're right. Ok so the weird thing is that one of my Admins pinged me last
night telling me about the issue, I checked it out and sure enough the
scroll bars wouldn't move.
But upon checking it just now, you are correct, there are no scroll bars.
In fact the admin screen itself is completely different from what I saw
last night.
When I looked last night, the option menu was on the right hand side of the
admin menu. It was the old menu we were looking at. In my mind it was
correct as I am accustomed to the old layout.
So my guess is that as the workshop folder on the server stores multiple
revisions of each mod, the server must have derped and loaded an old
version of Shine.
Everything is ok now, sorry about the confusion and thanks for checking
into it.
